src/components/Notification.js

```diff
@@ -37,7 +37,8 @@ export const Notification = (props) => {
         icon = <InfoIcon fill={'#3c763d'} style={ iconStyle } />;
         break;
     }
-    const content = <ReactMarkdown source={notificationData.message} linkTarget={'_blank'} className={'underlined'}/>;
+    // eslint-disable-next-line react/no-children-prop
+    const content = <ReactMarkdown children={notificationData.message} linkTarget={'_blank'} className={'underlined'}></ReactMarkdown>;
     notificationDiv = div({
```

Comment:
breaking change, change source to children

Reply:
My only feedback would be to lose the closing tag:
```
const content = <ReactMarkdown children={notificationData.message} linkTarget={'_blank'} className={'underlined'}/>;
```

Reply:
After that, 👍🏽